samples/zigbee/light_bulb/src/main.c

```diff
@@ -515,6 +528,12 @@ void main(void)
 	bulb_clusters_attr_init();
 	level_control_set_value(dev_ctx.level_control_attr.current_level);
 
+        /** Initialize ZCL scene table */
```

Comment:
I don't think we use `/**` for comments.
Also, use tabs instead of spaces.

Reply:
fixed